ocs_ci/ocs/pgsql.py

```diff
@@ -428,9 +417,7 @@ def respin_pgsql_app_pod(self):
             pod status
 
         """
-        app_pod_list = get_operator_pods(
-            constants.PGSQL_APP_LABEL, constants.RIPSAW_NAMESPACE
-        )
+        app_pod_list = get_operator_pods(constants.PGSQL_APP_LABEL, constants.BMO_NAME)
```

Comment:
the BMO_NAME is not define in constants.

Reply:
Done.